homeassistant/components/switch/rpi_rf.py

```diff
@@ -84,6 +86,10 @@ def __init__(self, hass, name, rfdevice, protocol, pulselength,
         self._pulselength = pulselength
         self._code_on = code_on
```

Comment:
Thanks for the note @andrey-git, I forgot about that. I just changed it with the latest commit.